Source/URLEncodedFormEncoder.swift

```diff
@@ -930,8 +936,8 @@ final class URLEncodedFormSerializer {
     }
 
     func serialize(_ array: [URLEncodedFormComponent], forKey key: String) -> String {
-        var segments: [String] = array.map { component in
-            let keyPath = arrayEncoding.encode(key)
+        var segments: [String] = array.enumerated().map { index, component in
+            let keyPath = arrayEncoding.encode(key, index: index)
```

Comment:
Seems you've missed the use site for the updated name.

Reply:
Thanks for catching it, I forgot to build  🤦🏽‍♂️.